### DISCUSSION TITLE
How do Update Flight level

### DISCUSSION THREAD

**Discussion by Adi-18** [flybywiresim/aircraft#8363] [Help] [answered]
I have a flight plan from world view. It's a short fly, so my level has to be 5100 ft.
But MCDU shows FL 200. Absolutely wrong.
How can I reassign FL for my short fly? In [Init] its on FL051
![image](https://github.com/flybywiresim/aircraft/assets/29123730/315f07d1-8b30-43bc-918f-b2a7b786fae2)

**Comment by tracernz** [comment, marked as answer]
These are altitude constraints coming from the STAR you have selected (BLM3G). You can select a different STAR, or delete the constraints (although you will no longer be complying with the STAR if you do that).
